docs/source/example_notebooks/gcm_counterfactual_medical_dry_eyes.ipynb

```diff
@@ -0,0 +1,291 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Modeling of the graph\n",
+    "\n",
+    "We know that the Treatment node and the Condition node are causing the Vision, but we don't know (other than in the data generation process) the structural causal model. The latter, as well as the noise terms are recovered by the counterfactual algorithm, as long as the post-nonlinear assumption is not violated in the data. We assume that this graph represents the reality, and we assume that there are no hidden confounders (causal sufficiency). So, we fit the dataset we have to the graph. We are going to need this causal graph to answer any counterfactual querries later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import networkx as nx\n",
+    "import dowhy.gcm as gcm\n",
+    "\n",
+    "causal_model = gcm.InvertibleStructuralCausalModel(nx.DiGraph([('Treatment', 'Vision'), ('Condition', 'Vision')]))\n",
+    "gcm.auto.assign_causal_mechanisms(causal_model, medical_data)\n",
+    "\n",
+    "gcm.util.plot(causal_model.graph)\n",
+    "\n",
+    "gcm.fit(causal_model, medical_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now returning back to our original problem, let's load Alice's data, who happens to have rare allergy (Condition = 1)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specific_patient_data = pd.read_csv('newly_come_patients.csv')\n",
+    "specific_patient_data.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Answering Alice's counterfactual querries\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "In cases where we want to examine a hypothetical outcome if an event had not happened or if it had happened\n",
+    "differently, we employ the so called Counterfactual logic based on structural causal models. Thus, we examine what the value of the target node (Vision) would be, if the Treatment node had a different value (0: for no treatment, or 1: for option A drops), given that we know the outcome of one Treatment value (in this case we know the target Vision for Treatment 2: the option B eye drops that Alice got), and given that the node Condition is 1 based on the Alices history (Alice has reported that she has the rare allergy). To check these counterfactual querries someone needs the aforementioned causal graph and to provide her own data, as well as the value of the Treatment that she wants to examine."
```

Comment:
> Thus, we examine what the value of the target node (Vision) would be, if the Treatment node had a different value (0: for no treatment, or 1: for option A drops), given that we know the outcome of one Treatment value (in this case we know the target Vision for Treatment 2: the option B eye drops that Alice got), and given that the node Condition is 1 based on the Alices history (Alice has reported that she has the rare allergy). 

Suggest rewriting for clarity.  Maybe bullets can help?
* Given we know Alice's treatment was "option B" (Treatment == 2),
* and that she has the rare allergy (Condition==1),
* and that her vision was .88 (Vision == 0.883874)
* what Vision outcome would she taken a different treatment?  (either Treatment ==0 or Treatment == 1)

Instead of having "option A" and "option B" be treatment 1 and 2, why not call them "option 1" and "option 2"?

Reply:
Revised the paragraph and renamed "A" and "B" to "1" and "2".